src/components/GridItem.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Box,
+  Card,
+  CardContent,
+  CardMedia,
+  CardActionArea,
+  Grid,
+  Modal,
+  Typography,
+} from "@mui/material";
+// import { itemDetailsList } from "../helpers/ItemList";
+
+const modalStyle = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 400,
+  bgcolor: "background.paper",
+  border: "2px solid #000",
+  boxShadow: 24,
+  p: 4,
+};
+
+const menuContainerStyle = {
+  padding: 5,
+  margin: 5,
+};
+
+function GridItem({ itemDetailsList }) {
+  const [selectedItem, setSelectedItem] = useState(null);
```

Comment:
Lets destructure selectedItem to this 

`const { name, description, image, price } = selectedItem`

Apply this `menuItem` as well

Reply:
Destructured both selectedItem and menuItem.